src/anyon/qpu_interface.jl

```diff
@@ -145,7 +145,7 @@ function serialize_job(circuit::QuantumCircuit,repetitions::Integer)::String
         "circuit"=>Dict{String,Any}(
                 "operations"=>Vector{Dict{String,Any}}()
             ),
-        "num_repetitions"=>repetitions
```

Comment:
We should rename the variable `repetitions` to `shot_count` as well.

Reply:
The `repetitions` parameter in `serialize_job` looks like it's still there.

Reply:
Oops... good catch! Fixed now.